src/app.js

```diff
@@ -1 +1,32 @@
 'use strict';
+
+const { terminal } = require('./terminal');
+const { generateRandomNumber } = require('./RandonNumberGenerator');
+const { guess } = require('./Checker');
+const guessedNumber = generateRandomNumber();
+
+function startGame() {
+  terminal.question(
+    'Try to guess. Enter a number: ', (userInput) => {
+      if (userInput.length !== 4 || !Number(userInput)) {
+        console.log('Invalid input. Enter 4 different digits');
+        terminal.question('Try to guess. Enter a number: ', startGame);
```

Comment:
I think there must be a different phrase, kinda 'You have to enter a 4-digit number'?.. 
And you can type it in console.log and then just put startGame() in the next line.